docs/getting-started/quickstart.md

```diff
@@ -0,0 +1,185 @@
+---
+slug: quickstart
+append_help_link: true
+title: Scan your first project with Semgrep
+hide_title: false
+description: Learn how to set up Semgrep and scan your first project
```

Comment:
```suggestion
description: Learn how to set up Semgrep and scan your first project.
```
(To update if we change the term "project".)

Reply:
Changed this to repository based on your suggestion above.